frontend/src/tabs/zoom.pug

```diff
@@ -21,11 +21,25 @@
       v-bind:avgsize="info.avgCommitSize",
       v-bind:mergegroup="info.isMergeGroup")
 
+  .zoom__toggle-commit-message-body(v-if="info.user.commits.length")
+    button(v-if="expandedCommitMessagesCount === 0" v-on:click="toggleAllCommitMessagesBody(true)") show all commit messages body
+    button(v-else v-on:click="toggleAllCommitMessagesBody(false)") hide all commit messages body
+
   .zoom__day(v-for="day in info.user.commits")
     template(v-if="day.insertions > 0")
       h3(v-if="filterTimeFrame === 'week'") Week of {{ day.date }}
       h3(v-else) {{ day.date }}
       ul
-        li(v-for="slice in day.commitResults")
-          a(v-bind:href="getSliceLink(slice)", target="_blank") {{ slice.messageTitle }}
+        li(v-for="slice in day.commitResults").commit-message.active
+          a(v-bind:href="getSliceLink(slice)" target="_blank") {{ slice.messageTitle }}
           span &nbsp; ({{ slice.insertions }} lines)
+          a(
+            v-if="slice.messageBody.length > 0",
```

Comment:
Sorry for the last minute request, I think `slice.messageBody !== ''"` would be better in this case for better readability and consistency with line 33

Reply:
A gentle reminder that， there are two places for your previous clause。

Reply:
My bad. Thought it was only this line